README.md

```diff
@@ -130,3 +130,49 @@ following command:
 ​
 You should be able to see the file and potentially others stored in the same
 location.
+
+
+## Download Instructions
+
+This sections provides guidelines and intructions on how to download the data files. It contains three steps - Generate Public and Secret Key, Decrypt the data files and Validate the decrypt data files. 
+
+### Preparing the system
+
+Downloading data files requires [crypt4gh](https://www.ga4gh.org/news/crypt4gh-a-secure-method-for-sharing-human-genetic-data/)
+for decrypting the data files. 
+
+
+### Generate Public and Secret Key
+
+The first step is to generate a keypair using the `crypt4gh` encryption tool. This can be done using the following command. There are two keys generated - `user.sec` is the secret key and `user.pub` is the public key. You must reply to the FEGA-SE helpdesk by sending them the generated public key - `user.pub`. 
+
+```bash
+crypt4gh-keygen --sk user.sec --pk user.pub
+```
+
+### Decrypting the data files
+
+You recieve an email from the FEGA-SE helpdesk that contains a text file. This text file contains links to download the encrypted data files. Download the download_data script from [here](https://raw.githubusercontent.com/NBISweden/LocalEGA-SE-Deployment/master/dev-tools/publicize_file/download_data.sh?token=GHSAT0AAAAAABQNNF76ZH6NKQZFOOGELTXQYPOT6FA) and use the following command.
+
+```bash
+./download_data.sh path/filename
+```
+This downloads all the encrypted files from the text file and allows it to maintains the structure of the dataset. Next, you must transfer the downloaded encrypt files to a secure environment. Use the following command to decypt files inside the secure environment using the secret key - `user.sec` that was generated in the previous step.
```

Comment:
Maybe mention that the user needs to transfer the private key as well to the secure environment